src/components/PageSection/svg/CurvedSvg.tsx

```diff
@@ -56,17 +61,18 @@ export const CurvedSvgBottom = styled(CurvedSvg)<StyledSvgProps>`
   fill: ${({ svgFill, theme }) => svgFill || theme.colors.background};
 `
 
-const ConcaveContainer = styled(Box)<{ clipPath: string }>`
+const ConcaveContainer = styled(Box)<{
+  clipPath: string
+  concaveBackgroundLight?: string
+  concaveBackgroundDark?: string
+}>`
   width: 100%;
   height: 20px;
-  background-color: ${({ clipPath, theme }) => {
-    if (clipPath === '#topConcaveCurve') {
-      return '#7645d9'
```

Comment:
Removing hard coded colors - using props instead @Chef-Cheems 

Case could probably be made for the `concaveColors` being an object (maybe also being amalgamated with `concave`) rather than separate into two strings.